queries/nix/highlights.scm

```diff
@@ -82,39 +82,40 @@
 (interpolation "${" @punctuation.special (_) "}" @punctuation.special) @none
 
 ; fields (the `.` in `a.b = c;` isn't included)
-(attrset (bind . (attrpath (attr_identifier) @field)))
-(rec_attrset (bind . (attrpath (attr_identifier) @field)))
+(attrset_expression (binding_set (binding . (attrpath (identifier) @field))))
+(rec_attrset_expression (binding_set (binding . (attrpath (identifier) @field))))
 
 ; unary operators
-(unary "-" @operator)
-(unary "!" @operator)
+(unary_expression "-" @operator)
+(unary_expression "!" @operator)
 
 ; binary operators
-(binary "?" @operator)
-(binary "++" @operator)
-(binary "*" @operator)
-(binary "/" @operator)
-(binary "+" @operator)
-(binary "-" @operator)
-(binary "//" @operator)
-(binary "<" @operator)
-(binary "<=" @operator)
-(binary ">" @operator)
-(binary ">=" @operator)
-(binary "==" @operator)
-(binary "!=" @operator)
-(binary "&&" @operator)
-(binary "||" @operator)
-(binary "->" @operator)
+(binary_expression "++" @operator)
+(binary_expression "*" @operator)
+(binary_expression "/" @operator)
+(binary_expression "+" @operator)
+(binary_expression "-" @operator)
+(binary_expression "//" @operator)
+(binary_expression "<" @operator)
+(binary_expression "<=" @operator)
+(binary_expression ">" @operator)
+(binary_expression ">=" @operator)
+(binary_expression "==" @operator)
+(binary_expression "!=" @operator)
+(binary_expression "&&" @operator)
+(binary_expression "||" @operator)
+(binary_expression "->" @operator)
```

Comment:
Not change it to just the plain literals?

Reply:
Since I can't test, I prefer not to make any changes beyond those absolutely necessary to unblock the updates.